build.sbt

```diff
@@ -92,6 +92,6 @@ val api = Project("membership-attribute-service", file("membership-attribute-ser
     addCommandAlias("batch-load", "runMain BatchLoader"),
     addCommandAlias("play-artifact", "riffRaffNotifyTeamcity"),
   )
-  .dependsOn(`membership-common`)
+  .dependsOn(`membership-common` % "test->test;compile->compile")
```

Comment:
this is needed so we can access TestCatalog in membership common from the MDAPI tests.
https://www.scala-sbt.org/1.x/docs/Multi-Project.html#:~:text=A%20useful%20declaration,compile%2D%3Ecompile%22).